src/App.svelte

```diff
@@ -53,25 +55,32 @@
 </script>
 
 <main class="mainContainer">
-  <Auth bind:authResponse></Auth>
+  <Auth bind:this={authComponent} bind:authResponse bind:loggedOut></Auth>
   {#if authResponse}
     <div class="topBar">
       <h1>Trøkk</h1>
       <h2>Hei {authResponse.userInfo.givenName}!</h2>
-      <button on:click={() => openSettings = !openSettings} >Settings</button>
+      <div>
+        <button on:click={() => openSettings = !openSettings} >Instillinger</button>
+        <button on:click={() => authComponent.logout()} >Logg ut</button>
+      </div>
     </div>
     {#if openSettings}
       <Settings on:save={handleNewPaths}></Settings>
     {/if}
     <Files bind:scannerPath></Files>
+  {:else if loggedOut}
+      <div class="login">
+        <button on:click={() => authComponent.login()}>Logg in</button>
```

Comment:
```suggestion
        <button on:click={() => authComponent.login()}>Logg inn</button>
```
Litt norwenglish her